vcf/gVcf.go

```diff
@@ -110,7 +110,7 @@ func sampleToString(s Sample) string {
 			answer += fmt.Sprintf("%s%d", PhasedToString(s.Phase[i]), s.Alleles[i])
 		}
 	}
-	if len(s.FormatData) > 0 {
+	if len(s.FormatData) > 1 {
```

Comment:
Could you help me understand why this line of code is changed?

Reply:
Format data is a series of ':' delimited strings. if the format data is of length 1, it doesn't need to be joined by a ':', as only the genotype is in the sample field. Since the genotype is written separately earlier in this function, nothing from FormatData needs to be written out